README.md

```diff
@@ -45,6 +45,10 @@ Once drafting is complete and the anchor position is set, a plan can be placed i
 ### object methods
 #### Methods in draft mode (without anchor)
   - plan_obj:add_node(plan_pos, node)       - add a node to plan - if adjustment is given, the min/max and ground_y is calculated
+  - plan_obj:add_mts_obj(mts, pos, replacements)  - add MTS Object to nodes at optional position(defaults to `{x=0,y=0,z=0}`)
```

Comment:
For beter understanding I avoided to use "pos" in documentation. Please use "plan_pos", "world_pos" or "anchor_pos" instead. In this case it should be "plan_pos".